#! /usr/bin/env python

PACKAGE='aricc_2d_vision'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("h_high", int_t, 0, "The high value of H", 100, 0, 255)
gen.add("h_low",  int_t, 0, "The low value of  H", 1, 0, 255)

gen.add("s_high", int_t, 0, "The high value of S", 100, 0, 255)
gen.add("s_low",  int_t, 0, "The low value of  S", 1, 0, 255)

gen.add("v_high", int_t, 0, "The high value of V", 100, 0, 255)
gen.add("v_low",  int_t, 0, "The low value of  V", 1, 0, 255)
exit(gen.generate(PACKAGE, "aricc_2d_vision", "ColourFinder"))
